### PR TITLE
Auto sync with subscription events.

### PR DESCRIPTION
**Pull request by alrik**

### CODE REVIEW

File: assets/js/agent/app.vue [hunk 1]
```diff
@@ -56,27 +56,85 @@
...
     },
     methods: {
       loadChannels: function() {
         var $data = this.$data;
         var me = this;
 
         this.realmq.channels.list().then(function (channelList) {
-          channelList.items.sort(function (a, b) {
+          $data.channels = channelList.items.sort(function (a, b) {
             return new Date(a.createdAt) < new Date(b.createdAt) ? 1 : -1;
```
Comment (baranga) [inline assets/js/agent/app.vue]: Not part of this PR but its not necessary to cast to Date as iso date time strings are lexical sortable.

File: assets/js/agent/app.vue [hunk 2]
```diff
@@ -56,27 +56,85 @@
...
+          .then(function(channel) {
+            var existentChannel = channels.find(function(channel) {
+              return channel.id === channelId;
+            });
+
+            if (!existentChannel) {
+              existentChannel = me.getChannelViewModel(channel);
+              channels.unshift(existentChannel);
+            } else {
+              // todo update channel data.
```
Comment (baranga) [inline assets/js/agent/app.vue]: Something missing here?

Comment (alrik) [reply assets/js/agent/app.vue]: Not really.

Comment (baranga) [reply assets/js/agent/app.vue]: Either add a update of channel name or remove else branch completely.

Comment (alrik) [reply assets/js/agent/app.vue]: Yes: see 2f2e2f2

**Review by baranga — APPROVED** [review]
